Library/Homebrew/formula.rb

```diff
@@ -475,8 +473,9 @@ def inspect
   # less consistent and the standard parameters are more memorable.
   def std_cmake_args
     %W[
+      -DCMAKE_C_FLAGS_RELEASE=""
```

Comment:
Shouldn't need to pass `""`, these arguments are not parsed by the shell. Cmake may ignore them, but I don't think they're necessary.